### ISSUE TITLE
Ionic app error

### ISSUE THREAD

**Issue by ellisium** [AppsFlyerSDK/PhoneGap#34]
I encounter an error when launching your sdk on Ionic application. Same your documentation 👍 
`$ionicPlatform.ready(function() {      

    var options = {
           devKey:  'xxXXXXXxXxXXXXxXXxxxx8'// your AppsFlyer devKey               
         };

    if (ionic.Platform.isIOS()) {
        options.appId = "123456789";            // your ios app id in app store 
    }

      window.plugins.appsFlyer.initSdk(options);      
  });`
Error :
cordova.js:426 Wrong type for parameter "args" of AppsFlyer.initSdk: Expected Array, but got Object

**Comment by af-fess** [comment]
@ellisium yes, we did the change before a month+. Looks like you use old version of plugin. I suggest you to uninstall this plugin (see through `cordova plugin ls`) and install new [cordova-plugin-appsflyer-sdk](https://github.com/AppsFlyerSDK/cordova-plugin-appsflyer-sdk). This repo `PhoneGap` will be deprecated soon
